component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -866,7 +866,8 @@ public List<Property> getConfigurationProperties() {
         headers.setDisplayName("HTTP Headers");
         headers.setRequired(false);
         headers.setDescription("Enter the headers used by the API separated by comma, with the Header name and value " +
-                "separated by \":\". If the phone number and text message are in Headers, specify them as $ctx.num and $ctx.msg");
+                "separated by \":\". If the phone number and text message are in Headers, specify them as $ctx.num " +
+                "and $ctx.msg or $ctx.otp");
```

Comment:
Same as previously, shall we change the description to something like follows.
```suggestion
                "separated by \":\". If the phone number, text message, and OTP are in Headers, specify them as $ctx.num," +
                " $ctx.msg, and $ctx.otp");
```